src/utils/brand.utils.ts

```diff
@@ -0,0 +1,63 @@
+import { brandConfig } from "../constants/brand.constants";
+
+/**
+ * This regex will match any official deriv production and testing domain names.
+ * Allowed deriv domains: localhost, binary.sx, binary.com, deriv.com, deriv.be, deriv.me and their subdomains.
+ *
+ * @param {string} domainName - The path to be match with regex.
+ * @returns {string} Returns the boolean whether its valid/allowed URL.
+ */
+export const isDomainAllowed = (domainName: string) => {
```

Comment:
```suggestion
export const isDomainAllowed = (domainName: string) =>
    /^(((.*)\.)?(localhost:8443|pages.dev|binary\.(sx|com)|deriv.(com|me|be|dev)))$/.test(domainName);
```

Reply:
[optional] you can remove `{}` and `return`

Reply:
@fasihali-deriv could u please move this to https://github.com/deriv-com/deriv-utils/blob/master/src/utils/url.utils.ts ? ( also its related test cases )